<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QDialog" name="setting">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(242, 242, 242, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="File_MainText">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>260</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/파일_50.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Alram_MainText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>12</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/img/알람설정_50.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #ffffff;
selection-color: #ffffff;
color: #ffffff;

border: 1px solid  #000000;
border-radius: 7px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: #F28B30;
	border: 1px solid  #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../../../Users/kjy19/OneDrive/Desktop/App7.3 JJINJJIN/resources/resources.qrc">
     <normaloff>:/img/img/초기화_35.png</normaloff>:/img/img/초기화_35.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>43</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: #ffffff;
selection-color: #ffffff;
color: #ffffff;

border: 1px solid  #000000;
border-radius: 7px;
padding-right: 5px; 
padding-left: 5px;
padding-top: 5px;
padding-bottom: 5px;
}

QPushButton:hover
{
	background-color: #F28B30;
	border: 1px solid  #000000;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../../../../Users/kjy19/OneDrive/Desktop/App7.3 JJINJJIN/resources/resources.qrc">
     <normaloff>:/img/img/확인_30.png</normaloff>:/img/img/확인_30.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>43</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>296</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="helmet_alram">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));

QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 30px;
    height: 30px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../../../../Users/kjy19/OneDrive/Desktop/App7.3 JJINJJIN/resources/resources.qrc">
        <normaloff>:/img/img/안전모 유무.png</normaloff>:/img/img/안전모 유무.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="rack_alram">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../../../../Users/kjy19/OneDrive/Desktop/App7.3 JJINJJIN/resources/resources.qrc">
        <normaloff>:/img/img/위험구역_35.png</normaloff>:/img/img/위험구역_35.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="car_alram">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../../../../../Users/kjy19/OneDrive/Desktop/App7.3 JJINJJIN/resources/resources.qrc">
        <normaloff>:/img/img/충돌.png</normaloff>:/img/img/충돌.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="logvideo_combox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>효율 저장 (저화질 압축으로 인한 영상 화질 저하 발생)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>고화질 (1920x1080)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>일반화질 (1280x720)</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../Users/kjy19/OneDrive/Desktop/App7.3 JJINJJIN/resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>setting</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>setting</receiver>
   <slot>Alram_setting_reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ok()</slot>
  <slot>Alram_setting_reset()</slot>
  <slot>Logvideo_path()</slot>
 </slots>
</ui>
